diff --git a/node_modules/haul/src/utils/makeReactNativeConfig.js b/node_modules/haul/src/utils/makeReactNativeConfig.js
index fd068de..b7cb33e 100644
--- a/node_modules/haul/src/utils/makeReactNativeConfig.js
+++ b/node_modules/haul/src/utils/makeReactNativeConfig.js
@@ -122,7 +122,9 @@ const getDefaultConfig = ({
       'process.env': {
         NODE_ENV: dev ? '"development"' : '"production"',
         DEV_SERVER_ORIGIN: JSON.stringify(`http://localhost:${port}`),
-      },
+        CART_API_URL: JSON.stringify(process.env.CART_API_URL),
+        LANG_CODE: JSON.stringify(process.env.LANG_CODE)
+      },  
       __DEV__: dev,
     }),
     new webpack.LoaderOptionsPlugin({
